extensions/amp-sidebar/1.0/sidebar.jss.js

```diff
@@ -22,14 +22,19 @@ const sidebarClass = {
   maxHeight: '100vh !important',
   height: '100vh',
   maxWidth: '80vw',
-  backgroundColor: '#efefef',
   minWidth: '45px !important',
   outline: 'none',
   overflowX: 'hidden !important',
   overflowY: 'auto !important',
   zIndex: 2147483647,
 };
 
+// User overridable styles
```

Comment:
We also need `box-sizing: border-box !important` in the `sidebarClass`

Reply:
Thanks added.